testsuite/testcases/basic/failure_oom_1x1.yaml

```diff
@@ -22,9 +22,9 @@ jobs:
               memory: 10Mi
               cpu: 100m
 ---
-timeout: "150s"
```

Comment:
This test now takes just over two and a half minutes in GitHub Actions (but it's much faster when I run it in isolation on my machine), so I've increased this timeout.